### ISSUE TITLE
Updating plugin on current stable neovim (0.3.7) causes it to silently fail

### ISSUE THREAD

**Issue by fluffywaffles** [mhartington/nvim-typescript#249]
# Problems summary

I'm on Arch Linux, and even I have a neovim version of only 0.3.8. According to [the homepage](https://neovim.io/), the current stable version is 0.3.7. As I regularly do, I updated my plugins and nvim-typescript simply stopped working. There is no error messaging. There was no error or deprecation warning issued on installation. No errors are reported when I open TypeScript files or try to issue nvim-typescript commands like `:TSDef`. It just breaks. I'm trying not to sound upset, but this is really pretty irritating.

Look, I'm happy to update to bleeding-edge 0.4.0 if you just warn me first. The problem here is that there was no messaging around this upgrade.

## Expected

The plugin should work with current stable neovim when installed as directed in the README. Rather than being forced to trace the issue all the way to the GitHub page, I should receive some kind of deprecation warning if my plugin decides to stop supporting my version of neovim. Issuing `:TS*` commands should log an error message using `echom` so that I can see what went wrong.

Basically don't just break without warning or explanation.

## Environment Information

 * terminal: any
 * vim/nvim version: current stable (0.3.7) and also on 0.3.8

## Provide a minimal vim rc with less than 50 lines (Required!)

I'm not doing this. It's not necessary.

**Comment by mhartington** [comment]
Closed via https://github.com/mhartington/nvim-typescript/commit/e7186f57d8ace5344e105e539f0bf1e1425754e7

**Comment by fluffywaffles** [comment]
:tada: thanks for the quick fix!
